src/config/quantTypes.json

```diff
@@ -384,6 +384,58 @@
       ]
     }
   },
+  "library10XVDJ": {
+    "key": "library10XVDJ",
+    "name": "Library - 10X VDJ - Tapestation",
+    "wellType": "TS10XVDJ",
+    "parse": {
+      "delimiter": ",",
+      "rowDelimiter": ["\r\n", "\r", "\n", "\r\r\n"],
+      "from": 2,
+      "columns": [
+        "filename",
+        "wellId",
+        "sampleDescription",
+        "fromBasePair",
+        "toBasePair",
+        "averageSizeBasePairs",
+        "concentration",
+        "regionMolarity",
+        "percentageOfTotal",
+        "regionComment"
+      ]
+    },
+    "metadata": {},
+    "conversion": {
+      "factors": {
+        "dilution": 10
+      },
+      "expression": "(dilution)",
+      "decimalPlaces": 3
+    },
+    "qcResults": {
+      "key": "molarity",
+      "units": "nM",
+      "assay": {
+        "type": "Library - X10 VDJ",
+        "version": "v1.0"
```

Comment:
I basically made these assay values up.  I have no idea if these are OK or not.

Reply:
Question for user. It is likely the name of a QC assay kit they use on the Tapestation system.